configure.ac

```diff
@@ -57,7 +57,7 @@ case "$host" in
 	*-mingw*)
 		CPPFLAGS="${CPPFLAGS} -DWIN32_LEAN_AND_MEAN"
 		CPPFLAGS="${CPPFLAGS} -D_WIN32_WINNT=NTDDI_WINXP"
-		LDFLAGS="${LDFLAGS} -Wl,--nxcompat"
+		LDFLAGS="${LDFLAGS} -Wl,--nxcompat,--dynamicbase,--export-all-symbols"
```

Comment:
Is this hack of adding --export-all-symbols still needed in mingw? There has to be a better solution than exporting all symbols